apps/give-cast/utils/rate.ts

```
@@ -0,0 +1,27 @@
+      method: "GET",
+      headers: {
+        accept: "application/json",
+        "content-type": "application/json",
+        "X-CMC_PRO_API_KEY": process.env.TICKER_API_KEY || "",
+      },
+    }
+    res = await fetch(url, opt)
+    tkr = await res.json()
+    usd = tkr?.data[coin]?.quote?.USD?.price || 0
+    console.warn("Ticker:", usd)
+  } catch (ex: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)